blocks/columns/columns.css

```diff
@@ -150,3 +150,140 @@
 }
 
 
+/* Two-column block styling */
+.section:has(.columns.two-column) h1#media-newsroom {
+  color:#6B7E7C;
+  font-size: 50px;
+  font-weight: 400;
+  text-transform: uppercase;
+  letter-spacing: 0.08em;
+  line-height: 1.2em;
+  margin-top: 30px;
+  margin-bottom: 30px;
+  text-align: left;
+  display: block;
+  padding-top: 80px;
+}
+
+/* Make the columns wrapper flex and position on right */
+.columns-wrapper:has(.columns.two-column) {
+  display: flex ;
+  justify-content: flex-end ; /* pushes content to right side */
+  align-items: flex-start;
+  width: 100%; 
+}
+
+/* Override default columns behavior - be very specific */
+.columns.two-column {
+  display: flex ;
+  flex-direction: row ;
+  width: 50% ;
+}
+
+/* Force individual columns to display side by side */
+/* stylelint-disable-next-line no-descending-specificity */
+.columns.two-column > div {
+  color: black ;
+  flex: 1 ;
+  display: flex ;
+  width: auto ;
+  font-size: 20px;
+  letter-spacing: 0.02em;
+  font-family: Antarctica, sans-serif;
+  line-height: 1.5em;
+  font-weight: 300;
+}
+
+/* First column paragraph - ONLY for two-column */
+.columns.two-column > div > div:nth-child(1) > p {
+  padding-right: 30px;
+  width: 400px;
+  text-align: left;
+  line-height: 1.5em;
+  padding-bottom: 20px;
+  letter-spacing: 0.02em;
+  font-family: Antarctica, sans-serif;
+  font-size: 20px;
+}
+
+/* Second column strong text - ONLY for two-column */
+.columns.two-column > div > div:nth-child(2) > p:nth-child(1) > strong{
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 1.5em;
+  letter-spacing: 0.02em;
+  font-family: Antarctica, sans-serif;
+}
+
+.columns.two-column >div > div:nth-child(2) > p.button-container > a.button:any-link,
+.columns.two-column > div> div:nth-child(2) > p.button-container > button{
+  color: #839790 ;
+  font-weight: normal ;
+  font-size: 20px ;
+  line-height: 1.5em;
+  letter-spacing: 0.02em;
+  font-family: Antarctica, sans-serif;
+  text-decoration: none ;
+  transition: colour 0.5s ;
+  position: relative;
+  display: inline-block;
+  border: none ;
+  outline: none ;
+  box-shadow: none ;
+  padding: 0 ;
+  background-color: white ;
+}
+
+
+
+/* Mobile styles */
+@media (width <= 600px) {
+  .section:has(.columns.two-column) h1#media-newsroom {
```

Comment:
rules should be in ">=" format.

`<=600px` tend to be mobile view - suggest to make this css mobile-first. Means all rules under `<=600px` become default mobile view, the other rules get added under `>=600px` and `>=900px` as needed